app/frontend/wallet/helpers/alertIfUnsupportedTrezorFwVersion.js

```diff
@@ -9,7 +9,7 @@ async function alertIfUnsupportedTrezorFwVersion() {
     const minSupportedFwVersion = '2.0.8'
     const {model, fwVersion} = await getTrezorInfo()
 
-    if (model === 'T' && fwVersion < minSupportedFwVersion) {
+    if (model === 'T' && versionCompare(fwVersion, minSupportedFwVersion) < 0) {
```

Comment:
`versionCompare` can return NaN, making `Nan < 0` resolve as false and not display the alert even tho something probably went wrong(?)

Reply:
fair point, and I would say that's the desired behavior in that case, so I would leave it as-is. This whole code is meant to cause as little hassle as possible if something goes wrong since it is basically a hotfix to notify people that their trezor version does not match which the trezor interface currently fails to state explicitly, only suggesting you to update your firmware without further explanation.